dashboard/app/models/concerns/concept_difficulties.rb

```diff
@@ -0,0 +1,20 @@
+module ConceptDifficulties
+  extend ActiveSupport::Concern
+
+  CONCEPTS = %w(
+    sequencing
+    debugging
+    repeat_loops
+    repeat_until_while
+    for_loops
+    events
+    variables
+    functions
+    functions_with_params
+    conditionals
```

Comment:
Not for now. We will want to do some kind of tagging later but we want to do a card sort (or similar) to figure out what our categories are first.